test/classic/install_test.go

```diff
@@ -0,0 +1,67 @@
+//go:build e2e
+
+package classic
+
+import (
+	"context"
+	"testing"
+
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/Dynatrace/dynatrace-operator/test/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/test/kubeobjects/pod"
+	"github.com/Dynatrace/dynatrace-operator/test/oneagent"
+	"github.com/Dynatrace/dynatrace-operator/test/sampleapps"
+	"github.com/Dynatrace/dynatrace-operator/test/setup"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+func install(t *testing.T) features.Feature {
+	installClassicFullStack := features.New("install classic fullstack")
+	secretConfig := getSecretConfig(t)
+
+	setup.InstallAndDeploy(installClassicFullStack, secretConfig, "../testdata/classic-fullstack/sample-deployment.yaml")
+	setup.AssessDeployment(installClassicFullStack)
```

Comment:
nitpick: So the csi-driver is not needed for classicfullstack, but that check makes sure that the csi-driver is alive.

I think the only e2e tests affected by the csi-driver should fail in case the csi-driver is problematic.
Also customers using classic probably doesn't even install the csi-driver.

Reply:
removed waiting for csi driver